eras/conway/impl/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 ### `testlib`
 
+* Add arbitrary functions to choose registration and unregistration certs in imp-tests. #4830
+  * `arbitraryRegTxCert` to choose between `RegTxCert` and `RegDepositTxCert`
+  * `arbitraryUnRegTxCert` to choose between `UnRegTxCert` and `UnRegDepositTxCert`
```

Comment:
Perhaps too verbose for CHANGELOG, why not add this explanation to the commit message instead? 

```suggestion
* Add `arbitraryRegTxCert` and `arbitraryUnRegTxCert`  #4830
```